site/en/docs/android/trusted-web-activity/lay-of-the-land/index.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: "layouts/doc-post.njk"
+title: Lay of the Land
+date: 2020-02-04
+updated: 2020-07-31
+description: An overview of the libraries and tools for creating Trusted Web Activities.
+authors:
+  - peconn
+---
+
+<div class="clearfix"></div>
```

Comment:
We probably don't need these on the site.

Reply:
Removed all of those.